### PR TITLE
No breaking change in setVerticesBuffer

### PR DESCRIPTION
**Pull request by benaadams**
Set https://github.com/BabylonJS/Babylon.js/pull/4026/files#r178512410

### CODE REVIEW

**Comment by deltakosh** [comment]
Thanks a lot!

File: src/Mesh/babylon.geometry.ts [hunk 1]
```diff
@@ -239,6 +239,11 @@
             if (kind === VertexBuffer.PositionKind) {
                 if (totalVertices != null) {
                     this._totalVertices = totalVertices;
+                } else {
+                    var data = <FloatArray>buffer.getData();
+                    if (data != null) {
+                        this._totalVertices = data.length / (buffer.byteStride * 4);
```
Comment (deltakosh) [inline src/Mesh/babylon.geometry.ts]: it is actually / 4 :)
I'll fix it

Comment (deltakosh) [reply src/Mesh/babylon.geometry.ts]: Lol

Comment (benaadams) [reply src/Mesh/babylon.geometry.ts]: https://github.com/BabylonJS/Babylon.js/pull/4069

**Pull request merged**